README.md

```diff
@@ -5,3 +5,114 @@ Arches for Science an Arches Application designed to help conservation scientist
+Navigate to the `arches-for-science` directory from your terminal and run:
+```
+pip install -e .
+```
+
+### If installing for deployment, run:
+```
+pip install arches-for-science
+```
+
+`Important`: Installing the arches-for-science app will install Arches as a dependency. This may replace your current install of Arches with a version from PyPi. If you've installed Arches for development using the `--editable` flag, you'll need to re pip install arches after installing arches-for-science
+
+## Project Configuration
+
+1. If you don't already have an Arches project, you'll need to create one by following the instructions in the Arches [documentation](http://archesproject.org/documentation/).
+Since Arches for Science uses `Cantaloupe` as its IIIF server, take notice of the
+Cantaloupe [installation instructions](https://arches.readthedocs.io/en/stable/developing/advanced/managing-and-hosting-iiif/), too.
+
+2. When your project is ready, add "arches_templating", "arches_for_science", and "pgtrigger" to INSTALLED_APPS **below** the name of your project:
+    ```
+    INSTALLED_APPS = (
+        ...
+        "my_project_name",
+        "arches_templating",
+        "arches_for_science",
+        "pgtrigger",
+    )
+    ```
+
+3. Make sure the following settings are added to your project
+    ```
+    FUNCTION_LOCATIONS.append("arches_for_science.functions")
+
+    TEMPLATES = build_templates_config(
+        debug=DEBUG,
+        app_root=APP_ROOT,
+        context_processors=[
+            "django.contrib.auth.context_processors.auth",
+            "django.template.context_processors.debug",
+            "django.template.context_processors.i18n",
+            "django.template.context_processors.media",
+            "django.template.context_processors.static",
+            "django.template.context_processors.tz",
+            "django.template.context_processors.request",
+            "django.contrib.messages.context_processors.messages",
+            "arches.app.utils.context_processors.livereload",
+            "arches.app.utils.context_processors.map_info",
+            "arches.app.utils.context_processors.app_settings",
+            "arches_for_science.utils.context_processors.project_settings",
+        ],
+    )
+
+    RENDERERS += [
+        {
+            "name": "xy-reader",
+            "title": "XY Data File Reader",
+            "description": "Use for all instrument outputs with x-y data",
+            "id": "e93b7b27-40d8-4141-996e-e59ff08742f3",
+            "iconclass": "fa fa-bolt",
+            "component": "views/components/cards/file-renderers/xy-reader",
+            "ext": "txt",
+            "type": "text/plain",   
+            "exclude": "",
+        },
+    ]
+
+    XY_TEXT_FILE_FORMATS = ["txt"]
+
+    X_FRAME_OPTIONS = "SAMEORIGIN"
+    ```
+
+4. Next ensure arches and arches_for_science are included as dependencies in package.json
+    ```
+    "dependencies": {
+        "arches": "archesproject/arches#dev/7.6.x",
```

Comment:
Should we note that in production installs people should probably point to stable branches?